ManualText.txt

```diff
@@ -920,32 +900,35 @@ Work orders for Physical Plant are handled online. When the officer requests you
-•	Enter 4444 as the phone number and Public Safety as the department.
+•	Scroll to the bottom of the drop down list to select “Police.”  - if you do not see a drop down list here, notify the ACDC so that your access level in the system can be adjusted.
+•	Enter 4444 as the phone number and Police as the department.
 •	Fill in any information provided by the officer.   As much as possible, entries should be word for word descriptions as provided by the officer.  If you are unsure of the wording, read it back to the officer before submitting.
 •	Click “Submit.”
 CTS
 Work orders for CTS usually only fall under the category of major network emergencies or outages or problems with campus phones or our cameras.  We do not have to enter work orders for individuals who call us with network problems.  They can be connected directly to CTS (x4500).
 
-Dispatchers may enter CTS work requests for issues involving campus phones and technology if instructed by the officer (as with Physical Plant work orders), which can be done by emailing CTS@gordon.edu.  Always use the dispatch email function for these emails so that they are sent from the publicsafety@gordon.edu email address rather than your personal email address.
+Dispatchers may enter CTS work requests for issues involving campus phones and technology if instructed by the officer (as with Physical Plant work orders), which can be done by emailing CTS@gordon.edu.  Always use the dispatch email function for these emails so that they are sent from the police@gordon.edu email address rather than your personal email address.
 
 8.2	Escort Log (App. Escort Log)
-In the event of an escort (10-14E), you need to log the information in your dispatch log and also on a separate escort log, containing other necessary information used for quality control purposes. The escort log can be found on a clipboard to the right of the main dispatch console. On your dispatch log, record the name of the caller, the time, and the code 10-14E under the situation field. In the escort log, put the time that the call came in (same as in your log), the name of the person requesting an escort, where he or she is going (from and to), the officer responding, his or her arrival time, and his or her clear time.
+In the event of an escort, you need to log the information in Report Exec and also on a separate escort log, containing other necessary information used for quality control purposes. The escort log can be found in a binder to the right of the main dispatch console. In Report Exec, record the name of the caller, the time, and "Assistance: Escort" as the call type. In the escort log, put the time that the call came in, the name of the person requesting the escort, where he or she is going (from and to), the officer responding, his or her arrival time, and his or her clear time.
```

Comment:
We are likely going to do away with the escort log once we are on report exec only.